CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 * Release RTDB Emulator v4.4.1: Bugfix for unreleased feature.
 * Fixes a bug when using `firebase-tools` to delete Firestore documents inside the Functions emulator (#2001).
 * Fixes a bug where `.runtimeconfig.json` files were not properly detected (#1836).
-* Fixes a bug where rules were sometimes read as blank after a file save (#1980).
\ No newline at end of file
+* Fixes a bug where rules were sometimes read as blank after a file save (#1980).
+* Fixes an issue where callable functions with authentication could not be emulated (#1475).
```

Comment:
Is this just adding support for authentication in callable functions? Not really fixing something that was broken? If so, maybe reword the message.

Reply:
This is actually a bugfix, I would consider the previous behavior wrong (this always worked in prod, emulators should be like prod unless otherwise specified)